### PR TITLE
3.4 List of open source events.

### LINKED ISSUE #192: OpenSouce  : 3.4 List of open source events .
**List of open source events**
**Link to their websites**
**How to participate in it**
Please assign this to me !

### PR DESCRIPTION
**Pull request by siddhi-244**
<hr>

## Description 📜

List of open source events and links to apply for it.
Fixes #192 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Open Source

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Open_Source/3.4_List_Of_Open_Source_Events.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## Here is a list of some great open source events.
```
Comment (Aryamanz29) [inline Open_Source/3.4_List_Of_Open_Source_Events.md:1]: Change heading

File: Open_Source/3.4_List_Of_Open_Source_Events.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Hacktoberfest
+Hacktoberfest is a really popular open-source program that
+is hosted by Digital Ocean and is open to everyone across the world.
+Here, you need to submit four or more quality pull requests and you’ll either
+get a chance to plant a tree or get a limited-edition t-shirt. There is no such selection criteria.
+So, this is very useful for individuals, in particular, who are just starting out with open source. 
+You can contribute to participating projects on GitHub and even if you’re not able to make a large major contribution what you
+can get out of it is how to use Git and Github.
+
```
Comment (Aryamanz29) [inline Open_Source/3.4_List_Of_Open_Source_Events.md:10]: Add Images after every program 👇

File: Open_Source/3.4_List_Of_Open_Source_Events.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+## GirlScript Summer of Code
+GirlScript Summer of Code is the 3 month-long Open Source program during summers conducted by GirlScript Foundation, started in 2018, with an aim to help beginners get started with Open Source 
+Development while encouraging diversity. Throughout the program, participants contribute to different projects under the guidance of experienced mentors. Top participants get exciting goodies and 
+opportunities.
+
+[Link to apply](https://gssoc.girlscript.tech/)
+
+<hr>
+
```
Comment (Aryamanz29) [inline Open_Source/3.4_List_Of_Open_Source_Events.md:68]: Remove Extra Lines ☝

**Review by Aryamanz29 — CHANGES_REQUESTED** [review]

**Comment by Aryamanz29** [comment]
@siddhi-244  Please do req changes and once it's done ping me

**Comment by siddhi-244** [comment]
Done @Aryamanz29

**Review by Aryamanz29 — APPROVED** [review]
LGTM😉
Thanks for contributing🎉🎉🎉

**Comment by siddhi-244** [comment]
> LGTM😉
> Thanks for contributing🎉🎉🎉

Thank you😊. You keep on motivating me.

**Pull request merged**